modules/prebidServerBidAdapter/index.js

```diff
@@ -246,6 +246,10 @@ function queueSync(bidderCodes, gdprConsent, uspConsent, s2sConfig) {
     payload.us_privacy = uspConsent;
   }
 
+  if (gppConsent) {
+    // TODO - need feedback on how to pass gpp values to payload for PBS usersync endpoints...
```

Comment:
committee feedback is: take the regs.gpp json object.

Reply:
between two options: stringify the json object or pass two strings, one for gpp and another for gpp_sid, server feedback prefers option 1.